packages/dropdownable/tests/__snapshots__/Dropdownable.test.tsx.snap

```diff
@@ -151,37 +151,14 @@ exports[`Dropdownable is visible after opening 2`] = `
       </ButtonBase>
     </PrimaryButton>
   </div>
-  <clickOutside(DropdownContents)
+  <clickOutside(undefined)
     isOpen={true}
   >
-    <DropdownContents
+    <ForwardRef
       isOpen={true}
     >
-      <ForwardRef
-        style={
-          {
-            "left": "0",
-            "position": "absolute",
-            "top": "0",
-            "zIndex": 2,
-          }
-        }
-      >
-        <Overlay
-          innerRef={[Function]}
-          overlayRoot={
-            <body>
-              <div>
-                <div
-                  style="z-index: 2; position: absolute; left: 0px; top: 0px;"
-                >
-                  <p>
-                    I am a touchdown
-                  </p>
-                </div>
-              </div>
-            </body>
-          }
+      <div>
```

Comment:
Looks like there is an extra div being added here do you know where that addition is coming from?